### PR TITLE
Start 'Real-time Networking' chapter

### PR DESCRIPTION
**Pull request by Sean-Der**
Covers the concepts that are shared between Media and Data

### CODE REVIEW

**Comment by Sean-Der** [comment]
@mogren what do you think at a high level? I will make sure to run this through grammarly, don't want you spending all your time on reviewing stuff :(

File: content/docs/05-real-time-networking.md [hunk 1]
```diff
@@ -4,4 +4,64 @@ type: docs
 weight: 6
 ---
 
-TODO
+# Why is networking so important in Real-time communication?
+The network is the limiting factor in Real-time communication. In an ideal world we would have unlimited bandwidth
+and packets would arrive instantaneously. This isn't the case though. Networks are limited, and the conditions can
+could change at anytime. Measuring the network is also a difficult problem. Networks perform differently in subtle
```
Comment (mogren) [inline content/docs/05-real-time-networking.md:10]: "can could" -> could

File: content/docs/05-real-time-networking.md [hunk 2]
```diff
@@ -4,4 +4,64 @@ type: docs
...
+ways.
+
+Real-time communication also poses a problem that doesn't exist in most other domains. For a web developer it isn't fatal
+if your website is slower on some networks. As long as all the data arrives users are happy. With WebRTC if your data is
```
Comment (mogren) [inline content/docs/05-real-time-networking.md:14]: Nit: With WebRTC, if...

File: content/docs/05-real-time-networking.md [hunk 3]
```diff
@@ -4,4 +4,64 @@ type: docs
...
+late it is useless. No one cares about what was said in a conference call 5 seconds ago.  So when developing a RTC systems
```
Comment (mogren) [inline content/docs/05-real-time-networking.md:15]: systems -> systems

File: content/docs/05-real-time-networking.md [hunk 4]
```diff
@@ -4,4 +4,64 @@ type: docs
...
+it comes down trade-offs. What is my time limit, and how much can I send?
+
+This chapter covers the concepts that apply to both data and media communication. In later chapters we go beyond
+the theoretical and discuss how WebRTC's media and data subsystems solve these problems.
+
+## What are the attributes of the network that make it difficult?
+Code that effectively works across all network can be very complicated. You have lots of different factors, and they
+all can affect each other subtly. These are the most common issues that developers will encounter
```
Comment (mogren) [inline content/docs/05-real-time-networking.md:23]: "they all can" or "they can all"?  Also, missing a "." after the sentence.

File: content/docs/05-real-time-networking.md [hunk 5]
```diff
@@ -4,4 +4,64 @@ type: docs
...
+
+#### Jitter
+Jitter is the fact that `Transmission Time` may vary. Some times you will see packets arrive in bursts.
+Any piece of hardware along the network path can introduce issues.
+
+#### Packet Loss
+Packet Loss is when messages are lost in transmission. The loss could be steady, or it could come in spikes.
+This is also a common issue, especially on satellite or Wi-Fi.
+
+#### Maximum transmission unit
```
Comment (mogren) [inline content/docs/05-real-time-networking.md:41]: Do you prefer to upper case each word in the titles? Especially if they are known acronyms.

File: content/docs/05-real-time-networking.md [hunk 6]
```diff
@@ -4,4 +4,64 @@ type: docs
...
+Maximum Transmission Unit is the limit on how large a single packet can be. Networks don't allow you to send
+one giant message. At the protocol level, you need to split your data into multiple small packets.
```
Comment (mogren) [inline content/docs/05-real-time-networking.md:43]: I hesitated on changing this earlier, but something still doesn't feel right with this sentence. Maybe something like:

At the protocol level, messages might have to be split into multiple smaller packets.

File: content/docs/05-real-time-networking.md [hunk 7]
```diff
@@ -4,4 +4,64 @@ type: docs
...
+use a protocol like [Path MTU Discovery](https://tools.ietf.org/html/rfc1191) to figure out the largest packet size you can send.
+
+### Congestion
+Congestion is when the limits of the network have been reached. This is usually because you have reached the peak
+bandwidth that the current route can handle. However, this could be hourly limits your ISP configures or anything
+a network operator wants to impose.
+
+Congestion exhibits itself in many different ways. There is no standardized behavior here though. In most
+cases when congestion is reached the network will drop excess packets. In other cases the network will buffer.
+This will cause the Transmission Time for your packets to increase. You could also may see more Jitter as your network becomes more congested.
```
Comment (mogren) [inline content/docs/05-real-time-networking.md:55]: Extra "may" not needed in this sentence.

**Review by mogren — COMMENTED** [review]
Argh, sorry, forgot to publish my comments earlier...

**Pull request merged**